src/main/java/nextstep/ladder/LadderGameController.java

```diff
@@ -1,19 +1,44 @@
 package nextstep.ladder;
 
 import nextstep.ladder.domain.LadderGame;
+import nextstep.ladder.domain.dto.LadderPlayerGameResult;
 import nextstep.ladder.domain.info.LadderGameInfo;
 import nextstep.ladder.view.InputView;
 import nextstep.ladder.view.ResultView;
 
 import java.util.List;
 
 public class LadderGameController {
-
     public static void main(String[] args) {
         List<String> playerNames = InputView.inputPlayers();
+        List<String> gameResults = InputView.inputGameResults();
         int ladderHeight = InputView.inputLadderHeight();
-        LadderGame ladderGame = LadderGame.of(LadderGameInfo.of(playerNames, ladderHeight));
-        ResultView.printResult(ladderGame.result());
+
+        LadderGame ladderGame = makeLadderGame(playerNames, gameResults, ladderHeight);
+        printLadderResult(ladderGame);
+
+        LadderPlayerGameResult gameResult = playLadderGame(ladderGame);
```

Comment:
```suggestion
        LadderPlayerGameResult gameResult = ladderGame.play();
```

으로 작성해주셔도 충분히 가독성이 뛰어나보이는데 분리하신 이유가 있을까요!?

Reply:
앗.. 이건 제가 미처 수정 못하고 올린거같아요..
이리 저리 테스트 해보았는데요.

실제로 하고 싶었던것은 이렇게 분리를 하고 싶었어요.
void - ladderGame.play() 게임을 실행 해라
something result - laggerGame.result() 게임 결과를 달라

이렇게 생각한것은 무언가를 return 해주는 함수는 builder인데 단순동사로는 return하는 무언가 있다라는 느낌이 들지 않는다고 생각이 들어서요. 저만의 생각일 수도 있지만 play가 더욱 그렇게 느껴져서요. 그래서 어떻게하면 return해주는 느낌이 들게 해볼 수 있을지 이리저리 해보다가.. 저 상태로 커밋이 되었네요 ㅠㅠ